docs/guide/web/default-theme.md

```diff
@@ -263,6 +263,44 @@ Customize labels used in the templates, useful for translating the interface for
     }
     ```
 
+### information <Badge text="added in v1.6.0" type="tip"/>
+
+Define the content of the navigation’s “information” block (see [nav](#nav) option above). This can be used to display a build date or the library version for example.
+
+```js
+{
+    information: [
+        {
+            label: 'Version',
+            value: require('./package.json').version,
+        },
+        {
+            label: 'Built on',
+            value: new Date(),
+            type: 'time', // Outputed as a <time /> HTML tag
+            format: (value) => {
+                return value.toLocaleDateString('en');
+            }
+        }
+    ],
+}
+```
+
+* **Default:**
+
+    ```js
+    [
+        {
+            label: config.labels.builtOn,
```

Comment:
can we link this to the labels configuration section? this might be confusing as to what it means.

Reply:
✅